### ISSUE TITLE
Implement Tokenmom Exchange

### ISSUE THREAD

**Issue by tokenmom** [coingecko/cryptoexchange#1222]
Implement Tokenmom Exchange
https://www.tokenmom.com
https://docs.tokenmom.com

**Comment by tokenmom** [comment]
can you let us know why tokenmom.com exchange is not implemented.
I can see other recent exchange issues are being implemented.
But our exchange is not until now.

**Comment by tokenmom** [comment]
Can you add our dev team github account for new pull request ?
https://github.com/zeus16
We need permission for new pull request
